scripts/setup.sh

```diff
@@ -12,10 +12,10 @@ if [[ ! -z $NVM_DIR ]]; then # skip if nvm is not available
   nvm install
 fi
 
-echo "Installing dependencies from Homebrew..."
-brew bundle --file=- <<EOF
-brew 'redis', restart_service: true
-EOF
+# echo "Installing dependencies from Homebrew..."
+# brew bundle --file=- <<EOF
+# brew 'redis', restart_service: true
+# EOF
```

Comment:
Why did you comment these lines out?

Reply:
Good catch! I accidentally committed this file.... https://github.com/artsy/force/pull/12102